pkg/pillar/cmd/nodeagent/nodeagent.go

```diff
@@ -17,19 +17,17 @@ package nodeagent
 
 import (
 	"bytes"
-	"flag"
 	"fmt"
+	"github.com/lf-edge/eve/pkg/pillar/cmd"
```

Comment:
This package name is confusing to me. Can we call it agentbase or something more specific than cmd?

Reply:
You want me to put agent.go in a newly created agentbase package, or do 

`agentbase "github.com/lf-edge/eve/pkg/pillar/cmd"`

Reply:
We need it in a new package - agentbase . I thought thats what we did??

Reply:
Added comments .. So we need to put these under pillar/agentbase/ .
The package name should be agentbase.